licensing/src/main/java/io/crate/license/License.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.license;
+
+import java.util.Objects;
+
+/**
+ * Data structure for CrateDB Licence
+ */
+public class License  {
+
+    private final long expiryDate;
+    private final String issuedTo;
+    private final String signature;
+
+    public License(long expiryDate, String issuedTo, String signature) {
```

Comment:
not sure if we want to include the License class already in this form. If so I'd consider either using a different type (e.g. `DateTime`) or include the unit in the name. (`expirationDateInMs`)

Reply:
I've renamed the field for now - it will indeed change thus I went for the quickest change for now